std/csv.d

```diff
@@ -39,6 +39,18 @@
  *         writefln("%s works as a %s and earns $%d per year",
  *                  record[0], record[1], record[2]);
  *     }
+ *
+ *     // the file with filename.csv would contain the same string as shown
+ *     // above, notice the byLine and joiner
+ *
+ *     auto file = File("filename.csv", "r");
+ *     foreach(record; csvReader!(Tuple!(string,string,int))(
+ *         file.byLine.joiner("\n")))
```

Comment:
I think it would look better if you put the `csvReader!(..)` at the end of the UFCS pipeline